enhancements/windows-containers/cluster-logging-windows-worker-nodes.md

```diff
@@ -31,144 +32,105 @@ status: implementable
-    containers to redirect logs to standard out from various sources like 
-    ETW, Perf Counters etc
-* Customization of log collection infrastructure
-  * We plan to support fluentd only
-  * Microsoft will provide us with a Windows fluentd binary or gem file
+* Development or customization of log collectors.
+  * We plan to use the Microsoft provided Windows Fluentd binary and the Fluentd
+    logging driver that enables Windows containers to read logs from various sources
+    using Fluentd input plugins and redirect them to various destinations using the 
+    Fluentd output plugins.
 
 
 ## Proposal
 
-The main idea here is to deploy fluentd as a log collection agent onto Windows 
-nodes after cluster logging operator is deployed onto the cluster. Fluentd has
-support for Windows logging. We plan to leverage it and run fluentd as a 
-Windows service.
+The main idea here is to deploy Fluentd as a log collection agent onto Windows 
+nodes after cluster logging operator is deployed on the OpenShift cluster. 
+Fluentd has support for Windows logging. We plan to leverage it and run Fluentd 
+as a Windows service.
 
 
 ### Implementation Details
 
-We plan to create a Windows Node logging Ansible Playbook that can
-
-* Download fluentd binary for Windows from a known location and transfers it to
-  the Windows node
-* Configure the container runtime to use fluentd as the logging driver
-* Configure fluentd to communicate with elasticsearch data store in the same 
-  format as logs emitted from linux worker nodes
-
-The inputs to this playbook
-
-* The ip address, port, certificate to connect the elasticsearch service
-* URL from which fluentd Windows binary can be downloaded from
-
-We can get the ipaddress and other details from various pods and secrets present
-in the OpenShift cluster, once the cluster logging operator has been deployed.
-If the certificates change, cluster admin should run the ansible playbook
-again. 
-
-Kubelet on Windows nodes logs to a file. So, we also need to configure 
-fluentd's record transformer to ensure that logs collected from the kubelet log
-file are in format understandable to elasticsearch
+WMCO image will be packed with the td-agent .msi installer binary in addition
+to the required binaries mentioned in WMCO enhancement.
+Once both WMCO and the Cluster Logging Operator are deployed on the cluster,
+following steps are required to configure logging:
+ * Install td-agent.
+    * td-agent is a stable distribution of Fluentd.
+ * Run td-agent's fluentdwinsvc as a Windows service.
+    * fluentdwinsvc is permanently registered as a Windows service by the .msi
+      installer in the latest version of td-agent.
+ * Install the required Fluentd input and output plugins onto Windows nodes.
+ * Configure the container runtime to use Fluentd as the logging driver.
+ * Configure td-agent to collect logs from various input sources and forward 
+   them to elasticsearch data storage. 
```

Comment:
Can all these steps be done from the CLI? I remember having issues with some MSI installers that require a GUI.

Can we build every component from source and does our build systems support building them? We cannot just grab binaries and throw them into the image.